cross/libtorrent/Makefile

```diff
@@ -1,17 +1,16 @@
 PKG_NAME = libtorrent
-PKG_VERS = 1.2.9
+PKG_VERS = 1.2.10
 PKG_EXT = tar.gz
 PKG_DIST_NAME = $(PKG_NAME)-rasterbar-$(PKG_VERS).$(PKG_EXT)
-PKG_DIST_SITE = https://github.com/arvidn/$(PKG_NAME)/releases/download/$(PKG_NAME)-$(PKG_VERS)
+PKG_DIST_SITE = https://github.com/arvidn/libtorrent/releases/download/$(PKG_NAME)-$(PKG_VERS)
 PKG_DIR = $(PKG_NAME)-rasterbar-$(PKG_VERS)
 
 BUILD_DEPENDS = native/python3
 # for boost python binding (BOOST_LIBRARIES = python), the dependency on cross/python3 must appear before cross/boost
 DEPENDS = cross/python3 cross/boost cross/geoip cross/libiconv cross/openssl
```

Comment:
Could this be dependent to python310?  Also OpenSSL is already a dependency of python so a repeat of it is probably unecessary (while being informative)

Reply:
Currently I have troubles with `cross/libtorrent` that is a _double target_ build. One builds the libtorrent and the other target builds the python-libtorrent wheel. And the prefered build tool is boost b2, ...

Yes I tried python310 but currently trying python38, as I got errors like:
```
configure: error: 
  Could not link test program to Python. Maybe the main Python library has been...
```
This error may be due to the fact, that it was never possible to build wheels in the cross folder.

The main challenge is to cross compile the wheel.  As there is no wheel source available neither for deluge-libtorrent nor python-libtorrent.

As I have fixed the python-wheel target name (was renamed from `build_python_wheel`to `build_python_wheel_target`)
The build now stops with:

```
/bin/sh: 1: Syntax error: "(" unexpected
make[3]: *** [../../mk/spksrc.python-wheel.mk:39: build_python_wheel_target] Error 2
```

but a second run of make in spk/deluge folder succeeds...

So I will try to use python310 first, and then fix the error above...

Reply:
> Could this be dependent to python310? Also OpenSSL is already a dependency of python so a repeat of it is probably unecessary (while being informative)

Finally it builds with python310. 
python is BUILD_DEPENDS only but openssl is a real dependency. - so it must not be removed.

BTW repeated dependencies are no issue, as those are skipped anyway (recognized by the cookie files).

And even when dependencies are already included by dependent packages, it is IMHO not recommended to remove _unnecessary_ dependencies for the sake of information of what is required by a package (and to avoid unexpected errors when dependencies of dependent packages change...)